ruby-challenges.rb

```diff
@@ -3,33 +3,116 @@
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns all the words that contain that particular letter.
 
-beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+# beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
-letter_o = 'o'
+ letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
 
-# -------------------2) Create a method that takes in an array of numbers and returns the sum of the numbers. Use the test variables provided.
+def beverage_sorter string
+    beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+    beverages_array.select {|item| item.include?(string)}
+    end  
+    
+    p beverage_sorter(letter_o)
+    p beverage_sorter(letter_t)
```

Comment:
Logic looks great, just make sure to read the prompt carefully, understanding all inputs needed.  The prompt mentions taking in both an array of words and a letter in the form of a string which means two arguments and two params.